docs/epiviz/getsettings.rst

```diff
@@ -38,23 +37,3 @@ First run EpiViz-AT.
 Then go to the Developer Console.
 This is a menu option in the web browser.
 Then type ``printSettingsJson()``. Copy to a file what that command prints.
```

Comment:
Should switch to describing the download button as it does the same thing as printSettingsJson and is much easier to use.